Dockerfile.ocp

```diff
@@ -3,7 +3,8 @@ FROM ubi8
 RUN yum update -y && \
     yum install -y openstack-ironic-api openstack-ironic-conductor crudini \
         iproute iptables dnsmasq httpd qemu-img parted gdisk ipxe-bootimgs psmisc procps-ng \
-        mariadb-server python2-chardet ipxe-roms-qemu && \
+        mariadb-server python2-chardet ipxe-roms-qemu \
+        python3-ironicclient python3-ironic-inspector-client python3-openstackclient && \
```

Comment:
Which version of OSC are we going to get here? The no-auth logic in 3.18.0 (or its dependencies) is broken.

Reply:
since we're installing from beta, the version will be indeed 3.18.0

Reply:
Yeah from the (now working, yay!) build job we see:

```
python3-ironic-inspector-client     noarch3.5.0-0.20190313131319.9bb1150.el8ost       openstack-beta-rhel8 66 k
 python3-ironicclient                noarch2.7.2-0.20190529060404.266a700.el8ost       openstack-beta-rhel8 451 k
 python3-openstackclient             noarch3.18.0-0.20190312140834.6868499.el8ost      openstack-beta-rhel8 1.1 M
```

Can we backport the bugfix to the downstream branch to fix this?

Reply:
the version included in OSP16 (current downstream and soon upstream too) is 3.19.0, so should be fine for 4.3

Reply:
That "python2-chardet" looks suspicious. Do we have a python3 version? How is it being used?